src/legacy/core_plugins/console/public/np_ready/application/containers/editor/legacy/console_editor/editor.tsx

```diff
@@ -34,17 +32,12 @@ import { applyCurrentSettings } from './apply_editor_settings';
 
 import { useSendCurrentRequestToES, useSetInputEditor } from '../../../../hooks';
 
+import * as senseEditor from '../../../../models/sense_editor';
 // @ts-ignore
-import { initializeEditor } from '../../../../../../../public/quarantined/src/input';
-// @ts-ignore
-import mappings from '../../../../../../../public/quarantined/src/mappings';
+import mappings from '../../../../../lib/mappings/mappings';
 
 import { subscribeResizeChecker } from '../subscribe_console_resize_checker';
-import { loadRemoteState } from './load_remote_editor_state';
-
-export interface EditorProps {
-  previousStateLocation?: 'stored' | string;
-}
+import { SenseEditor } from '../../../../models/sense_editor';
```

Comment:
Looks like this file is already being imported above? Can we consolidate the imports somehow?

Reply:
Looks like an auto-import :), will remove